test/tools/test_component_tool.py

```diff
@@ -345,37 +369,6 @@ def test_component_tool_in_pipeline(self):
         assert "Vladimir" in tool_message.tool_call_result.result
         assert not tool_message.tool_call_result.error
 
-    @pytest.mark.skipif(not os.environ.get("OPENAI_API_KEY"), reason="OPENAI_API_KEY not set")
```

Comment:
this test should not be removed
see #8913

Reply:
Good catch!